### PR TITLE
Skipping mgpu in sgpu

### PR DESCRIPTION
**Pull request by zstreet87**
Skipping multi-gpu test in rocm.bazelrc file for single gpu runs - needed for Navi

### CODE REVIEW

File: tensorflow/tools/tf_sig_build_dockerfiles/devel.usertools/gpu.bazelrc [hunk 1]
```diff
@@ -78,7 +78,7 @@ build:rbe --config=rbe_linux_cuda
 test:pycpp_filters --test_tag_filters=-no_oss,-oss_excluded,-oss_serial,-benchmark-test,-v1only,gpu,-no_gpu,-no_gpu_presubmit,-no_cuda11,-no_rocm
 test:pycpp_filters --build_tag_filters=-no_oss,-oss_excluded,-oss_serial,-benchmark-test,-v1only,gpu,-no_gpu,-no_gpu_presubmit,-no_cuda11,-no_rocm
 test:pycpp_filters --test_lang_filters=cc,py --test_size_filters=small,medium
-test:pycpp --config=pycpp_filters -- //tensorflow/... -//tensorflow/python/integration_testing/... -//tensorflow/compiler/tf2tensorrt/... -//tensorflow/core/tpu/... -//tensorflow/lite/... -//tensorflow/tools/toolchains/...
+test:pycpp --config=pycpp_filters -- //tensorflow/... -//tensorflow/python/integration_testing/... -//tensorflow/compiler/tf2tensorrt/... -//tensorflow/core/tpu/... -//tensorflow/lite/... -//tensorflow/tools/toolchains/... -//tensorflow/dtensor/python/tests:multi_client_test_2gpus -//tensorflow/dtensor/python/tests:multi_client_test_nccl_2gpus -//tensorflow/python/distribute/experimental:multi_worker_mirrored_strategy_test_2gpus
```
Comment (i-chaochen) [inline tensorflow/tools/tf_sig_build_dockerfiles/devel.usertools/gpu.bazelrc:82]: hmmm....I'm not sure is a good idea to put bazelrc to skip these two. I think maybe should skip these three into https://github.com/ROCm/tensorflow-upstream/blob/develop-upstream/tensorflow/tools/ci_build/linux/rocm/run_gpu_single.sh 


cc @jayfurmanek

Comment (zstreet87) [reply tensorflow/tools/tf_sig_build_dockerfiles/devel.usertools/gpu.bazelrc:82]: I couldn't get it to work in the `if` block in the script.  I mentioned that in a meeting and @jayfurmanek mentioned to look into modifying the bazelrc file instead.

**Review by i-chaochen — APPROVED** [review]
I think it's ok as run_gpu_multi.sh uses `nonpip_multi_gpu` config

**Comment by i-chaochen** [comment]
Retest Ubuntu-CPU please.
Retest Ubuntu-GPU-multi please.
Retest Ubuntu-GPU-single please.

**Comment by i-chaochen** [comment]
Retest Ubuntu-GPU-multi please.
Retest Ubuntu-GPU-single please.

**Comment by i-chaochen** [comment]
Retest Ubuntu-GPU-multi please.

**Comment by i-chaochen** [comment]
Retest Ubuntu-GPU-multi please.

**Comment by i-chaochen** [comment]
Retest Ubuntu-GPU-multi please.

**Comment by i-chaochen** [comment]
Retest Ubuntu-GPU-multi please

**Comment by i-chaochen** [comment]
Retest Ubuntu-GPU-multi please

**Comment by i-chaochen** [comment]
Retest Ubuntu-GPU-multi please

**Comment by i-chaochen** [comment]
@zstreet87 this is always failed at `//tensorflow/python/distribute:collective_all_reduce_strategy_test_xla_2gpu` in CI, could you have a quick check on your local as well to see whether is ok?

**Comment by zstreet87** [comment]
branch: `r2.15-rocm-enhanced`

command: `tf-docker /tensorflow > bazel --bazelrc=/usertools/rocm.bazelrc test --config=rocm --run_under=//tensorflow/tools/ci_build/gpu_build:parallel_gpu_execute -- //tensorflow/python/distribute:collective_all_reduce_strategy_test_xla_2gpu`

result: `//tensorflow/python/distribute:collective_all_reduce_strategy_test_xla_2gpu PASSED in 54.0s`

Is the CI using only 1 GPU?

**Comment by i-chaochen** [comment]
oh, now all tests are PASSED. You can merge now.

**Pull request merged**